pkg/rebalancer/storage/controller/config/config.go

```diff
@@ -18,11 +18,69 @@
 package config
 
 import (
+	"strings"
+
 	"github.com/vdaas/vald/internal/config"
 )
 
 type GlobalConfig = config.GlobalConfig
 
+type RebalanceReason uint8
+
+const (
+	BIAS RebalanceReason = iota
+	RECOVERY
+	MANUAL
+)
+
+func (r RebalanceReason) String() string {
+	switch r {
+	case BIAS:
+		return "bias"
```

Comment:
maybe deviation ?

Reply:
thank you
I fixed: https://github.com/vdaas/vald/pull/1187/commits/1c5f4a241d8b711fd215cf4c189f93b752113c73